<launch>
	<!-- vim: set ft=xml noet : -->
	<!-- udp://127.0.0.1:14551@14555 -->

  <!--mavros params-->
  <arg name="fcu_url" default="udp://127.0.0.1:14551@" />
  <arg name="gcs_url" default="udp://@localhost" />
  <arg name="tgt_system" default="1" />
  <arg name="tgt_component" default="1" />
  <arg name="log_output" default="screen" />
  <arg name="fcu_protocol" default="v2.0" />
  <arg name="respawn_mavros" default="true" />		<!-- penting -->
 
  
 	<env name="ROSCONSOLE_CONFIG_FILE" value="$(find uav_commander)/config/rosconsole.config" />
	<env name="ROSCONSOLE_FORMAT" value="[${severity}] [${time:%H:%M:%S}]: ${message}" />
  
  
  <include file="$(find mavros)/launch/node.launch">
    <arg name="pluginlists_yaml" value="$(find uav_commander)/config/apm_pluginlists.yaml" />
    <arg name="config_yaml" value="$(find uav_commander)/config/apm_config.yaml" />

    <arg name="fcu_url" value="$(arg fcu_url)" />
    <arg name="gcs_url" value="$(arg gcs_url)" />
    <arg name="tgt_system" value="$(arg tgt_system)" />
    <arg name="tgt_component" value="$(arg tgt_component)" />
    <arg name="log_output" value="$(arg log_output)" />
    <arg name="fcu_protocol" value="$(arg fcu_protocol)" />
    <arg name="respawn_mavros" default="$(arg respawn_mavros)" />
  </include>
  
	<node name="gazebo_things" pkg="gazebo_things" type="t265" output="screen">
		<param name="noise" value="true" />
		<param name="noise_max" value="0.2" />
		<param name="scale_max" value="0.1" />
		<param name="noise_z_max" value="0.4" />
		<param name="scale_z_max" value="0.2" />
		<param name="rf_depan" value="true" />
		<param name="rf_bawah" value="true" />
        <!-- RF SAMPING-->
		<param name="rf_kanan" value="true" />
		<param name="rf_kiri" value="true" />		
        
        <param name="tembok_x" value="6.8" />
		<!--<param name="bawah_topic" value="/mavros/distance_sensor/rangefinder_sub" />-->
    <param name="test_rf" value="true" />
    <param name="rf_atas" value="true" />
    <param name="link_name" value="iris::drone_with_camera::iris::base_link" />
  </node>
	<include file="$(find vision_to_mavros)/launch/sitl_tf_to_mavros.launch"></include>  

</launch>
